deployment/clowdapp.yml

```diff
@@ -302,6 +302,58 @@ objects:
           AND playbook_run_systems.updated_at < TIMESTAMP 'today'
         ORDER BY remediation_id;
 
+- apiVersion: metrics.console.redhat.com/v1alpha1
+  kind: FloorPlan
+  metadata:
+    name: remediations-hms
+  spec:
+    database:
+      secretName: ${FLOORIST_DB_SECRET_NAME}
+    objectStore:
+      secretName: ${FLOORIST_HMS_BUCKET_SECRET_NAME}
+    logLevel: ${FLOORIST_LOGLEVEL}
+    suspend: ${{FLOORIST_SUSPEND}}
+    disabled: ${{FLOORIST_DISABLED}}
+    queries:
+    - prefix: hms_analytics/remediations/remediations
+      chunksize: 50000
+      query: >-
+        SELECT r.id,
+               r.tenant_org_id as org_id,
+               r.created_at,
+               r.updated_at,
+               r.auto_reboot,
+               r.archived,
+               ri.issue_id,
+               ris.system_id,
+               ris.resolved
```

Comment:
@rexwhite If a resolved state is changed, does it touch the updated timestamp of its remediation plan? Neither issue nor issue_system have timestamps.

Reply:
I don't think the `updated` timestamp is updated when the resolved state is changed

Reply:
The resolved state is changed by remediations-copnsumer, which is using Knex as its ORM, and Knex doesn't automatically update those fields.  I'll see if there's a convenient way to update `updated_at` semi-manually...

Reply:
I think it would be good to add timestamps to the remediation_issue_systems table. Let me know if that makes sense, and I'll create a ticket for it.

Reply:
That sounds like a good idea to me